src/requirements/WalletActivity/WalletActivityRequirement.tsx

```diff
@@ -36,30 +36,30 @@ const WalletActivityRequirement = (props: RequirementProps): JSX.Element => {
     if (maxAmount && minAmount === undefined)
       return (
         <>
-          {`Have a wallet created before `}
+          {`Have a transaction before `}
           <DataBlockWithDate timestamp={maxAmount} />
         </>
       )
 
     if (maxAmount === undefined && minAmount)
       return (
         <>
-          {`Have a wallet created after `}
+          {`Have a transaction after `}
```

Comment:
Since this is wallet age, maybe this would be more correct?
```suggestion
          {`Have your first transaction after `}
```
(And the other ones similarly)